components/Shipping/models.rst

```diff
@@ -0,0 +1,186 @@
+Models
+======
+
+Rule
+----
+
+**Rule** object represents additional restrictions which shippable object have to fulfil in order to be support by given
+**ShippingMethod**.
+Rule has the following properties:
+
++---------------+------------------------------------------------+
+| Property      | Description                                    |
++===============+================================================+
+| id            | Unique id of the rule                          |
++---------------+------------------------------------------------+
+| type          | Reference to constant from RuleInterface       |
++---------------+------------------------------------------------+
+| configuration | Additional restriction which have to be fulfil |
++---------------+------------------------------------------------+
+| method        | Reference to ShippingMethod                    |
++---------------+------------------------------------------------+
+
+.. note::
+    This model implements the :ref:`component_shipping_model_rule-interface`. |br|
+    For more detailed information go to `Sylius API Rule`_.
+
+.. _Sylius API Rule: http://api.sylius.org/Sylius/Component/Shipping/Model/Rule.html
+
+Shipment
+--------
+
+**Shipment** object has methods to represent behaviors which have place during process of shipment.
```

Comment:
`[...] has methods to represent the events that take place during the process of shipment.`